README.md

```diff
@@ -73,9 +90,10 @@ See [history](HISTORY.md) for latest updates and important/breaking changes.
 This node module is inspired by the C program [romcal](http://www.romcal.net/) written by Kenneth G. Bath. This module, while exhibiting similar output, is written ground up using different tools and technologies and exposes many new functionalities.
 
 Additional credits for bug fixes, localizations and suggestions go to:
-- [@jarosz](https://github.com/jarosz)
+
 - [@emagnier](https://github.com/emagnier)
 - [@tukusejssirs](https://github.com/tukusejssirs)
+- [@jarosz](https://github.com/jarosz)
```

Comment:
I would replace this with link to `AUTHORS.md`.

Reply:
Done!